src/installation/liveimages/guide-chroot.md

```diff
@@ -0,0 +1,388 @@
+# Installation Guide (chroot)
+
+## Pre-installation
+
+### Set the keyboard layout
+
+The default console keymap is US. Available layouts can be listed with:
+
+```
+$ ls /usr/share/kbd/keymaps/*/*/*.map.gz | less
+```
+
+To modify the layout, append a corresponding file name to
+[`loadkeys(1)`](https://man.voidlinux.org/loadkeys), omitting path and file
+extension. For example, to set the Dvorak keyboard layout:
+
+```
+# loadkeys dvorak
+```
+
+### Verify the boot mode
+
+Many new motherboards will support booting in UEFI mode, but some will not,
+instead booting in legacy BIOS mode. You will need to know in which mode
+your system will boot when it is time to
+[partition the disks](#partition-disks).
+
+If you have boot via UEFI mode, the directory `/sys/firmware/efi` should
+exist and be populated.
+
+```
+$ ls /sys/firmware/efi
+```
+
+If you know your system supports booting into UEFI mode, but does not appear to
+have done so, check your UEFI/BIOS settings (can usually be accessed by pressing
+a key like `F12` early in the boot process. Look for settings like *UEFI-mode*
+or *Disable Legacy BIOS mode*.
+
+### Connect to the Internet
```

Comment:
There is already a network configuration section.

Reply:
Good call. I'll remove the section and instead reference that.